usr/share/rear/restore/BLOCKCLONE/default/400_restore_clone.sh

```diff
@@ -48,7 +48,11 @@ case "$(basename ${BLOCKCLONE_PROG})" in
         # Have "bs=1M" before BLOCKCLONE_PROG_OPTS because when BLOCKCLONE_PROG_OPTS
         # contains already e.g. "bs=4k" (cf. doc/user-guide/12-BLOCKCLONE.adoc)
         # the last of the two "bs=..." settings wins (at least with 'dd' on openSUSE Leap 15.1)
-        dd bs=1M $BLOCKCLONE_PROG_OPTS of=$BLOCKCLONE_SOURCE_DEV if=$backuparchive
+	if [[ -z  "$BLOCKCLONE_PROG_COMPRESS" ]] ; then
+            dd bs=1M $BLOCKCLONE_PROG_OPTS of=$BLOCKCLONE_SOURCE_DEV if=$backuparchive
+	else
+            $BLOCKCLONE_PROG_COMPRESS --decompress $backuparchive | dd bs=1M $BLOCKCLONE_PROG_OPTS of=$BLOCKCLONE_SOURCE_DEV
```

Comment:
Just a thought: ATM you have hard coded call semantics (`--decompress PATH_TO_FILE`), but configurable `BLOCKCLONE_PROG_COMPRESS`, which feels a bit like it can mislead users into using a different compression program without realizing it won't work.

Also, have you validated this change outside of the RPI scenario, which according to the docs doesn't even use `rear recover`?